Root logger configuration (#227)
microsoft/ApplicationInsights-Python: azure-monitor-opentelemetry-distro/README.md

* service_name = Specifies the [service][service_semantic_convention_doc] name.
* service_namespace = Specifies the [service][service_semantic_convention_doc] namespace.
* service_instance_id = Specifies the [service][service_semantic_convention_doc] instance id.
* disable_logging = If set to `True`, disables collection and export of logging telemetry. Defaults to False.
* logging_level = Specifies the [logging level][logging_level] of the Opentelemetry Logging Handler. Defaults to logging.NOTSET.

> * logging_root = Specifies the [root logger name][logger_name_hierarchy_doc] underwhich all logging will be instrumented. Defaults to the root logger of the logging hierarchy.

Reviewer: ```suggestion
* logger_name= Specifies the [root logger name][logger_name_hierarchy_doc] in which all to collect telemetry for. Defaults to the root("") logger.
```

* service_instance_id = Specifies the [service][service_semantic_convention_doc] instance id.

> * disable_logging = If set to `True`, disables collection and export of logging telemetry.

Reviewer: ```suggestion
* disable_logging = If set to `True`, disables collection and export of logging telemetry. Defaults to False.
```
```suggestion
* disable_logging = If set to `True`, disables collection and export of logging telemetry.
```

> * logging_level = Specifies the [logging level][logging_level] of the Opentelemetry Logging Handler. Ex: logging.WARNING.

Reviewer: ```suggestion
* logging_level = Specifies the [logging level][logging_level] of the Opentelemetry Logging Handler. Defaults to logging.NOTSET.
```
```suggestion
* logging_level = Specifies the [logging level][logging_level] of the Opentelemetry Logging Handler. 
```

> * logging_root = Specifies the [root logger name][logger_name_hierarchy_doc] underwhich all logging will be instrumented.

Reviewer: ```suggestion
* logger_name= Specifies the [root logger name][logger_name_hierarchy_doc] in which all to collect telemetry for. Defaults to the root("") logger.
```
```suggestion
* logging_root = Specifies the [root logger name][logger_name_hierarchy_doc] underwhich all logging will be instrumented.
```
